conf/battle/client.conf

```diff
@@ -174,6 +174,12 @@ macro_detection_punishment_time: 0
 // Set to 0 to disable
 macrochecker_delay: 600000
 
-// Should companion pets/elementals/homunculi/mercenaries display the guild emblem of their master? (Note 1)
-// Default: no
-companion_show_guild_emblem: no
\ No newline at end of file
+// Should companion pets/elementals/homunculi/mercenaries display the guild emblem of their master? (Note 3)
+// Uses unit types bit field values:
+// 0: None - Hide guild emblems for all companions
+// 4: Pets
```

Comment:
What each number means is already covered by says (Note 3).
It would be sufficient to just list to which types it applies in a single line.

Reply:
but elemental isnt covered, so should we add there too ?

// Note 3: Value is a bit field. If no description is given,
//         assume unit types (1: Pc, 2: Mob, 4: Pet, 8: Homun, 16: Mercenary)

Reply:
Feel free to add elemental (512 is correct) at the top of the file, but please remove the list here.